src/lib/rest/CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ SET (SOURCES
     orionReply.cpp
     OrionError.cpp
     HttpStatusCode.cpp
+    curl_context.cpp
```

Comment:
Not sure if this is the right style... we have also CamelCase (e.g. OrionError.cpp) and camelcase (e.g. orionReply.cpp). A little messy :) maybe we should talk about what we want (or what the style guide says) about this by skype in the common chat.